irma/ftp/sftp.py

```diff
@@ -53,7 +53,12 @@ def _connect(self):
             self._conn.window_size = pow(2, 27)
             self._conn.packetizer.REKEY_BYTES = pow(2, 32)
             self._conn.packetizer.REKEY_PACKETS = pow(2, 32)
-            self._conn.connect(username=self._user, password=self._passwd)
+            if self._auth == 'key':
+                pkey = RSAKey.from_private_key_file('/home/irma/.ssh/id_rsa')
```

Comment:
what do you mean by "all accounts are available" ? To be accepted, the public key must be explicitly set in <user>/.ssh/authorized_keys